imap_processing/idex/idex_l1b.py

```diff
@@ -350,15 +353,15 @@ def get_spice_data(
     # Get spacecraft spin phase in degrees
     spin_phase = get_spacecraft_spin_phase(query_met_times=met)
     imap_spin_phase = get_spin_angle(spin_phase, degrees=True)
-    # Get position and velocity of IMAP in ecliptic frame
+    # Get the position and velocity of IMAP in ecliptic frame
     ephemeris = imap_state(et, observer=SpiceBody.SUN)
-    # Get Idex pointing in the j2000 equatorial frame
+    # Get Idex pointing in the defined frame
     idex_pointing = instrument_pointing(
-        et, SpiceFrame.IMAP_IDEX, SpiceFrame.J2000, cartesian=True
+        et, SpiceFrame.IMAP_IDEX, IDEX_EVENT_REFERENCE_FRAME, cartesian=True
     )
     solar_lon = solar_longitude(et, degrees=True)
-
-    range_ra_and_dec = cartesian_to_spherical(idex_pointing)
+    # longitude and latitude
+    lon_and_lat = cartesian_to_spherical(idex_pointing)
```

Comment:
This is a bit misleading because it looks like it also has "r" as the first index. Maybe `spherical_coords = ...` or `func()[:, 1:]` to drop the "r" component.